build.gradle

```diff
@@ -18,11 +19,20 @@ repositories {
 dependencies {
     compile 'com.squareup:javapoet:1.12.1'
     compile 'net.fabricmc:tiny-mappings-parser:0.2.2.14'
-    compile 'com.google.guava:guava:28.2-jre'
+    runtimeOnly 'com.google.guava:guava:28.2-jre'
 
     compile 'org.ow2.asm:asm:8.0'
     compile 'org.ow2.asm:asm-analysis:8.0'
     compile 'org.ow2.asm:asm-commons:8.0'
     compile 'org.ow2.asm:asm-tree:8.0'
     compile 'org.ow2.asm:asm-util:8.0'
+
+    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.1.0'
+    testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.1.0'
+}
+
+test {
+    useJUnitPlatform()
 }
+
+tasks.assemble.dependsOn shadowJar
```

Comment:
is the shadow jar needed? shouldnt the deps be pulled in via maven using it in yarn?

Reply:
I just use the shadow jar so that grabbing it into a local directory is way easier.